.github/workflows/release.yml

```diff
@@ -0,0 +1,70 @@
+name: Build Releases
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+
+jobs:
+  release:
+    runs-on: ${{ matrix.os }}
+    strategy:
+        matrix:
+            os: [ubuntu-latest, windows-latest, macos-latest]
+    steps:
+      - name: "Create filename"
+        id: fn
+        shell: bash
+        run: echo "FILENAME=build/rctab_${{ github.ref_name }}_${{ runner.os }}.zip" >> $GITHUB_OUTPUT
+          
+      - uses: actions/checkout@v3
+
+      - name: "Set up JDK 17.0.2"
+        uses: actions/setup-java@v3
+        with:
+          java-version: '17.0.2'
+          distribution: 'temurin'
```

Comment:
When we update the JDK, is it as simple as incrementing it here and GH will take care of that? Also, do we ever have to change this `temurin` thing?

Reply:
1. Yep, we just increment it here
2. Not sure about temurin

Reply:
Apparently temurin is an anagram of "runtime", and it's associated with the Eclipse version of the JDK. Interesting. I guess we just keep this as-is.